server/schedulers/hot_region_config.go

```diff
@@ -30,6 +30,15 @@ import (
 	"github.com/unrolled/render"
 )
 
+const (
+	// NonePriority indicates there is no dim priority for hot-region-scheduler
+	NonePriority = ""
```

Comment:
I think that it might be disturbing for users to think that None means no priority, meaning that each dimension has the same priority

Reply:
I think our current implement already has priority, it looks:
```
read-priorities [key, byte]
write-priorities [btye, key]
```
am I right?

Reply:
updated.